src/MAUI/Maui.Samples/Samples/Data/EditFeaturesUsingFeatureForms/EditFeaturesUsingFeatureForms.xaml.cs

```diff
@@ -0,0 +1,160 @@
+                        // Check if the service geodatabase can apply edits
+                        if (serviceGeodatabase.ServiceInfo.CanUseServiceGeodatabaseApplyEdits)
+                        {
+                            // Apply edits to the service geodatabase
+                            await serviceGeodatabase.ApplyEditsAsync();
+                        }
+                        else
+                        {
+                            // Apply edits to the service feature table
+                            await serviceFeatureTable.ApplyEditsAsync();
+                        }
+                    };
+
+                    var cancelButton = new Button
+                    {
+                        Text = "Cancel",
+                    };
+
+                    cancelButton.Clicked += async (s, e) => await Shell.Current.Navigation.PopModalAsync();
+
+                    // Create a grid to hold the feature form view and the buttons
+                    var grid = new Grid();
+                    grid.RowDefinitions.Add(new RowDefinition { Height = new GridLength(1, GridUnitType.Star) });
+                    grid.RowDefinitions.Add(new RowDefinition { Height = GridLength.Auto });
+
+                    // Add the feature form view to the first row
+                    Grid.SetRow(featureFormView, 0);
+                    grid.Children.Add(featureFormView);
+
+                    // Create a grid for the buttons
+                    var buttonGrid = new Grid();
+                    buttonGrid.ColumnDefinitions.Add(new ColumnDefinition { Width = new GridLength(1, GridUnitType.Star) });
+                    buttonGrid.ColumnDefinitions.Add(new ColumnDefinition { Width = new GridLength(1, GridUnitType.Star) });
+
+                    // Add the save button to the first column
+                    Grid.SetColumn(saveButton, 0);
+                    buttonGrid.Children.Add(saveButton);
+
+                    // Add the cancel button to the second column
+                    Grid.SetColumn(cancelButton, 1);
+                    buttonGrid.Children.Add(cancelButton);
+
+                    // Add the button grid to the second row
+                    Grid.SetRow(buttonGrid, 1);
+                    grid.Children.Add(buttonGrid);
+
+                    // Create a ContentView to hold the grid
+                    var featureFormContentView = new ContentView
+                    {
+                        Content = grid,
+                        Padding = new Thickness(20),
+                        BackgroundColor = Color.FromArgb("#80000000"),
+                    };
+
+                    // Display the ContentView in a modal
+                    var modalPage = new ContentPage
+                    {
+                        Content = featureFormContentView,
+                        Title = "Edit Feature",
+                    };
+
+                    await Shell.Current.Navigation.PushModalAsync(modalPage);
+                }
```

Comment:
Why is this not done in xaml? Would simplify the sample a lot just declaring the UI in xaml and could just be a panel that is made visible rather than using modals.